internal/core/api/styled_text.rs

```diff
@@ -0,0 +1,533 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
+
+#[derive(Clone, Debug, PartialEq)]
+#[allow(missing_docs)]
```

Comment:
I think the warning is valid and if we commit this as is, we risk to forget to document.
I suggest making it 
`#![cfg_attr(feature="experimental-rich-text", allow(missing_docs))]`
that way, we would get the warning when we remove the experimental of it.

(Or we just add the docs now)